content/en/about/community/partners/index.md

```diff
@@ -23,6 +23,7 @@ Istio benefits from an ecosystem of great partners. Check 'em out and learn how
     {{< company_logo link="https://www.huaweicloud.com" logo="./huawei.png" alt="Huawei Cloud" >}}
     {{< company_logo link="https://www.ibm.com/cloud/" logo="./ibm-cloud.svg" alt="IBM Cloud" tagline="Built for all your applications. AI ready. Secure to the core." >}}
     {{< company_logo link="https://github.com/knative/docs" logo="./knative.png" alt="Knative" >}}
+    {{< company_logo link="https://layer5.io/" logo="https://raw.githubusercontent.com/layer5io/layer5/master/assets/images/layer5/layer5-tag-community-white-bg.png" alt="Layer5" tagline="The Service Mesh Community - characterizing and contrasting service mesh projects and related technologies" >}}
```

Comment:
@ashiscs I just noticed one more thing that would be good to change before merging. Notice that all the other entries have a copy of the image in the current directory, instead of a full URL. Could you add a copy of the image in this PR and then change the entry to use it. Also the tagline is a little long. Since "The Service Mesh Community" is already part of the image, how about change like this?
```suggestion
    {{< company_logo link="https://layer5.io/" logo="./layer5-tag-community-white-bg.png" alt="Layer5" tagline="Characterizing and contrasting service mesh projects and related technologies" >}}
```